### DISCUSSION TITLE
Customizable drug effects...

### DISCUSSION THREAD

**Discussion by littlesera** [littlesera/LSCG#243] [Ideas]
Allow to add custom drug keywords and effects...
* maybe this is sets of bindings that would slowly be applied, using something similar to MBS?
* drink consumer would need to look up any custom keywords on the drink crafter, meaning they need to still be in the same room.
* Would work on both injections and drink
* examples could be... living latex, dronification nanobots, teeentacllllees...

**Comment by jimmyG1** [comment]
this seems fun

**Reply by jimmyG1** [reply]
what wood the nanobots do mite i ask

**Reply by littlesera** [reply]
This feature set is probably very similar to #265 
Maybe make both work with just different delivery (filled glass vs potion bottle/wand)
"Any sufficiently advanced technology is indistinguishable from magic"

**Reply by anchillabc** [reply]
This one has a lot of potential I think, but may need more fleshing out. To me, this appeals a lot more than the magic wand idea - that one seems a bit more confrontational (similar to the net gun) and this one seems to have potential for more varied roleplay.

**Comment by jimmyG1** [comment]
> This feature set is probably very similar to #265 Maybe make both work with just different delivery (filled glass vs potion bottle/wand) "Any sufficiently advanced technology is indistinguishable from magic"

you cood use the injector as a way to put the nanos in
